docs/elements.md

```diff
@@ -347,6 +350,76 @@ Attribute | Type | Default | Description
 
 -----
 
+### `pl-order-blocks` element
+
+Element to arrange given blocks of code or text that are displayed initially in the *input area*. The blocks can be moved to the *output area* to construct the solution of the problem.  In the example below, the input area is denoted by the header "Drag from here" and the output area is denoted with the header "Construct your solution here".
+
+#### Sample element
+
+![](elements/pl-order-blocks.png)
+
+**question.html**
+
+```html
+<pl-question-panel>
+<p> How can we compute $c$, the summation of two given numbers $a$ and $b$?</p>
+</pl-question-panel>
+<pl-order-blocks answers-name="sum_numbers" >
+  <pl-answer correct="false">$a * b$</pl-answer>
+  <pl-answer correct="true">$a + b$</pl-answer>
+  <pl-answer correct="false">$a + c$</pl-answer>
+  <pl-answer correct="false">$b + c$</pl-answer>
+</pl-order-blocks>
+```
+
+
+#### Customizations
+
+Attribute | Type | Default | Description
+--- | --- | --- | ---
+`answers-name` | string | — | Variable name to store data in.
+`weight` | integer | 1 | Weight to use when computing a weighted average score over all elements in a question.
+`shuffle-options` | boolean | false | Specify whether the ordering of the blocks in the input area should be shuffled. If false, the ordering of the answers in the HTML file (defined by `<pl-answer>`) will be used.
+`max-incorrect` | integer | special | The maximum number of incorrect answers to be displayed in the input area. The incorrect answers are set using `<pl-answer correct="false">`. Defaults to displaying all incorrect answers.
+`min-incorrect` | integer | special | The minimum number of incorrect answers to be displayed in the input area. The incorrect answers are set using `<pl-answer correct="false">`. Defaults to displaying all incorrect answers.
+`permutation-mode` | string | "html-order" | One of the following: `any`, `html-order`, `ranking`. See details below for description.
+`check-indentation` | boolean | false | Whether indentation is checked for correctness.
+`external-grader` | boolean | false | Whether an external grader is used to determine the correct answers. See more details below.
+`file-name` | string | user_code.py  | Name of the file where the information from the blocks will be saved, to be used by the external grader.
+`leading-code` | string | — | The file name of the leading code, to be used by the external grader.
+`trailing-code` | string | — | The file name of the trailing code, to be used by the external grader.
```

Comment:
@nicknytko @mfsilva22 do we actually need these, or can they be handled by external grader options? The `pl-file-editor` doesn't have options like this, so I don't understand why they are needed here?

Reply:
@mwest1066 we need these files, at least the leading-code option, to insert for example the import calls. We use this same thing for workspaces. When using `pl-file-editor`, a similar thing is accomplished by using `input-code`.

Reply:
Discussed further on Slack. As @nicknytko suggested, I think we should leave this sort of code modification to the grader and remove these options from this element.